pkg/controller/installation/marketplace/manager.go

```diff
@@ -41,7 +43,7 @@ func GetOperatorSources() *operatorSources {
 
 //go:generate moq -out MarketplaceManager_moq.go . MarketplaceInterface
 type MarketplaceInterface interface {
-	CreateSubscription(ctx context.Context, serverClient pkgclient.Client, owner ownerutil.Owner, os marketplacev1.OperatorSource, ns string, pkg string, channel string, operatorGroupNamespaces []string, approvalStrategy coreosv1alpha1.Approval) error
+	InstallOperator(ctx context.Context, serverClient pkgclient.Client, owner ownerutil.Owner, os marketplacev1.OperatorSource, t Target, operatorGroupNamespaces []string, approvalStrategy coreosv1alpha1.Approval) error
 	GetSubscriptionInstallPlan(ctx context.Context, serverClient pkgclient.Client, subName, ns string) (*coreosv1alpha1.InstallPlan, *coreosv1alpha1.Subscription, error)
```

Comment:
This works, as in we need to wait for the InstallPlan to be sure the CRDs are in the cluster. What we really want to do to sequence things correctly is to wait for when the CSV is ready.

The subscription has a `status.currentCSV` prop and a CSV has a `status.phase` = Succeeded which should be able to be used to to this